sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+
+import time
+import argparse
+import asyncio
+import os
+import logging
+from collections import defaultdict
+
+from azure.identity.aio import ClientSecretCredential
+from azure.eventhub.aio import EventHubConsumerClient
+from azure.eventhub import EventHubSharedKeyCredential
+from azure.eventhub.extensions.checkpointstoreblobaio import BlobCheckpointStore
+from azure.eventhub import TransportType
+
+from logger import get_logger
+from process_monitor import ProcessMonitor
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--link_credit", default=3000, type=int)
+parser.add_argument("--output_interval", type=float, default=1000)
+parser.add_argument("--duration", help="Duration in seconds of the test", type=int, default=30)
+parser.add_argument("--consumer_group", help="Consumer group name", default="$default")
+parser.add_argument("--auth_timeout", help="Authorization Timeout", type=float, default=60)
+parser.add_argument("--offset", help="Starting offset", default="-1")
+parser.add_argument("--partitions", help="Number of partitions. 0 means to get partitions from eventhubs", type=int, default=0)
+parser.add_argument("--recv_partition_id", help="Receive from a specific partition if this is set", type=int)
+parser.add_argument("--track_last_enqueued_event_properties", action="store_true")
+parser.add_argument("--load_balancing_interval", help="time duration in seconds between two load balance", type=float, default=10)
+parser.add_argument("--conn_str", help="EventHub connection string",
+                    default=os.environ.get('EVENT_HUB_PERF_32_CONN_STR'))
+parser.add_argument("--eventhub", help="Name of EventHub")
+parser.add_argument("--address", help="Address URI to the EventHub entity")
+parser.add_argument("--sas_policy", help="Name of the shared access policy to authenticate with")
+parser.add_argument("--sas_key", help="Shared access key")
+parser.add_argument(
+    "--transport_type",
+    help="Transport type, 0 means AMQP, 1 means AMQP over WebSocket",
+    type=int,
+    default=0
+)
+parser.add_argument("--parallel_recv_cnt", help="Number of parallelling receiving", type=int)
+parser.add_argument("--proxy_hostname", type=str)
+parser.add_argument("--proxy_port", type=str)
+parser.add_argument("--proxy_username", type=str)
+parser.add_argument("--proxy_password", type=str)
+parser.add_argument("--aad_client_id", help="AAD client id")
+parser.add_argument("--aad_secret", help="AAD secret")
+parser.add_argument("--aad_tenant_id", help="AAD tenant id")
+parser.add_argument("--storage_conn_str", help="conn str of storage blob to store ownership and checkpoint data")
+parser.add_argument("--storage_container_name", help="storage container name to store ownership and checkpoint data")
+parser.add_argument("--uamqp_debug", help="uamqp logging enable", action="store_true")
```

Comment:
change to uamqp_logging_enable to have consistent names?

Reply:
updated